### DISCUSSION TITLE
how to access the value of my App.xaml file from my Platforms for Windows

### DISCUSSION THREAD

**Discussion by Eremmmm** [dotnet/maui#21023] [Q&A] [answered]
Hi, I would like to know how to access the value of my App.xaml file from my Platforms for Windows folder from my C# code to modify them? 
![image](https://github.com/dotnet/maui/assets/49162612/404a5c0c-f676-4bd1-9ff7-d8c3e2931a00)

**Comment by dush135** [comment]
I am also searching for an answer

**Reply by Eremmmm** [reply]
Hi, sorry I didn't dig deeper into the subject since my application concerns a test project.

**Comment by cgp1976** [comment, marked as answer]
There is a static class for it. look into MauiWinUIApplication

var thickness= MauiWinUIApplication.Current.Resources["TextControlBorderThemeThickness"] as Microsoft.UI.Xaml.Thickness;

**Reply by Eremmmm** [reply]
Ok, thanks
